media-libs/libplacebo/libplacebo-1.7.0.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2018 Gentoo Foundation
```

Comment:
PS: I recommend using `repoman commit`. Among basic QA checks and adding your sign-off, it would have auto-fixed this copyright header.

Reply:
I don't understand, what's wrong about this copyright header. `repoman scan` reports no issues for me. (But thanks for the tip!)

Reply:
wrong it is not, but since signoff requirement, repoman commit will auto-fix old headers to `Gentoo Authors` instead.